keras_hub/src/utils/preset_utils.py

```diff
@@ -155,7 +155,9 @@ def get_file(preset, path):
                 f"version). Received: preset={preset}"
             )
         try:
-            return kagglehub.model_download(kaggle_handle, path)
+            return kagglehub.model_download(
+                kaggle_handle, path, force_download=True
```

Comment:
let us not set the `force_download` to True

Reply:
Whoops, wasn't meant to be committed. Removed